python/minimal_connection.py

```diff
@@ -62,7 +57,7 @@ def main():
 
     for obj in tStream:
         print obj
-        pass
+#        pass
```

Comment:
Up to you.  It's nice to have an example user of the module but even examples should have debug/dead code removed from them.